index.html

```diff
@@ -55,5 +56,265 @@ <h2>I'm a Software Developer</h2>
+            </li>
+            <li>
+              <button type="button">Ruby</button>
+            </li>
+          </ul>
+        </nav>
+
+        <button type="button" class="postbtn2">See Project</button>
+      </nav>
+      <nav class="post">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <nav class="post-btn">
+          <ul>
+            <li>
+              <button type="button">html</button>
+            </li>
+            <li>
+              <button type="button">bootstrap</button>
+            </li>
+            <li>
+              <button type="button">Ruby</button>
+            </li>
+          </ul>
+        </nav>
+
+        <button type="button" class="postbtn2">See Project</button>
+      </nav>
+    </section>
+    <section class="about">
+      <div class="post-aboutme">
+        <h1>About me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button type="button">Get My Resume</button>
+      </div>
+
+      <div class="skills">
+        <div id="line2"></div>
+        <div>
+          <h2><span id="diamond"></span> Language</h2>
+        </div>
+        <h3>Javascript</h3>
+        <h3>Ruby</h3>
+        <h3>Html</h3>
+        <h3>Css</h3>
+        <div>
+          <h2><span id="rectangle"></span>Frameworks</h2>
+        </div>
+        <h3>Bootstrap</h3>
+        <h3>Ruby on Rails</h3>
+        <h3>RSPec</h3>
+        <h3>CapyBara</h3>
+        <h3>Selenium</h3>
+        <div>
+          <h2><span id="circle"></span> Skills</h2>
+        </div>
+        <h3>Codekit</h3>
+        <h3>Github</h3>
+        <h3>Codepen</h3>
+        <h3>Gitlab</h3>
+        <h3>Terminal</h3>
+      </div>
+    </section>
+    <footer>
+      <div>
+        <p class="foot">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </div>
+      <div>
+        <form action="https://formspree.io/f/mrgjlwqn" method="POST">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            name="textarea"
+            id="textarea"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <div>
+            <button class="footer-button" type="submit">Get in touch</button>
+          </div>

```

Comment:
- [ ] Kindly ensure that the effects for hover, active (pressed), and disabled are implemented for your button as we have in the template.

  ![buttons-color](https://user-images.githubusercontent.com/36542986/159686757-ccdaa84f-375c-46df-abff-7b4f29f3147f.jpg)